moteur-server-rest/routes.py

```diff
@@ -63,7 +63,8 @@ def handle_kill():
 def handle_status(workflow_id):
     document_root = get_env_variable("WORKFLOWS_ROOT")
     current_user = get_env_variable("USER")
-    check_process_command = f"ps -fu {current_user} | grep {workflow_id}/workflow.xml | grep -v grep"
+    workflow_file_name = get_env_variable('WORKFLOW_FILE_NAME')
```

Comment:
You may as well add 'workflow.json` as default as in the other places this env is necessary, although i do not like the repetition.
The best would be to add a `get_workflow_filename` in config.json that would handle that.

Reply:
I expected a new method in `config.py` to avoid duplicating the default `workflow.json` at several places.